src/components/SurveyQuestion.js

```diff
@@ -0,0 +1,47 @@
+const SurveyQuestion = ({
+  question,
+  options,
+  selection,
+  isMultiSelect,
+  onOptionClick,
+  onNextQuestion,
+  onPreviousQuestion,
+  onFinish,
+  isLastQuestion,
+  isSingleSelect,
+}) => {
+  return (
+    <div>
+      {/* <h1>{question}</h1> */}
+      <ul className="space">
+        {options.map((option) => (
+          <li key={option.label}>
+            <button
+              className={
+                (isSingleSelect && option.lable) ||
+                (isMultiSelect && option.lable)
+                  ? "check"
+                  : "selected"
+              }
+              onClick={() => onOptionClick(option.label)}
+            >
+              {option.label}
+            </button>
+          </li>
+        ))}
+      </ul>
+      <div className="button-container">
+        <button className="prev" onClick={onPreviousQuestion}>
```

Comment:
Unused class name, please remove.

Reply:
☝️ I see that this was not fixed yet.

Reply:
I already did this, I might not push my change maybe

Reply:
Push it then 😄 you can see it updated here, and then you can resolve the conversation

Reply:
I did empty commit, I really hope I already did a change, but forgot to push

Reply:
I still see the unused class name here.